pay/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+
+## 1.1.1 (2023-02-02)
+* Update `pay_android` to 1.1.0, which includes the latest version of the `flutter_svg` package.
```

Comment:
This bit is missing:
Update `pay_android` to 1.0.10, which...

Reply:
Fixed